docs/source/examples/examples.myst

```diff
@@ -7,5 +7,6 @@ maxcut-demo.myst
 Understanding probabilistic error cancellation (PEC) <pec-tutorial.myst>
 Zero noise extrapolation on IBMQ backends<ibmq-backends.myst>
 Zero noise extrapolation on PyQuil parametric programs <pyquil_demo.myst>
+vqe-demo_custom-gate.myst
```

Comment:
If you add a title next to the file name it should show up nicely like the other ones, I think. 
"Variational Quantum Eigensolver improved with Mitiq". 

Also, we'd better find a common choice for all files to either have hyphens or underscore. Can you pick one for this example? For example 
```
Variational Quantum Eigensolver improved with Zero Noise Extrapolation <vqe-demo-custom-gate>
```
I'd drop the 'with Mitiq', as this is already the Mitiq library documentation so it does not add much information.

Reply:
All good points, updated accordingly.